FlowCryptUI/Nodes/TableViewController.swift

```diff
@@ -10,9 +10,37 @@ import AsyncDisplayKit
 
 @MainActor
 open class TableNodeViewController: ASDKViewController<TableNode> {
+    public override var title: String? {
+        didSet {
+            navigationItem.setAccessibility(id: title)
+        }
+    }
     
     public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
         node.reloadData()
     }
 }
+
+public extension UINavigationItem {
+    func setAccessibility(id: String?) {
```

Comment:
Can this be non-optional? It's better to require it then to default to empty string.

Reply:
In current implementation we observe `didSet` event for viewcontroller title which can be nil. 
https://developer.apple.com/documentation/uikit/uiviewcontroller/1621364-title

If we want to make it non nil then we need to add this call to each viewcontroller implicitly

Reply:
```
public override var title: String? {
        didSet {
            navigationItem.setAccessibility(id: title)
        }
    }
```

Reply:
I don't fully understand the technicalities, but: every view has a title. Therefore every view could also have a title-based accessibilityId. Therefore there shouldn't be a need for optionality.

Reply:
Not every view has a title only where we set it implicitly.

Reply:
You meant explicitly. Ok, in that case it's fine.

Reply:
sorry, explicitly.